test/index.test-d.ts

```diff
@@ -141,6 +141,14 @@ const runServer = async () => {
   // access all errors
   app.post('/upload/files', async function (req, reply) {
     const { FilesLimitError } = app.multipartErrors
+
+    // test instanceof Error
+    const a = new FilesLimitError();
```

Comment:
Can you use a `tsd` assertion?

Reply:
It is used to test if an error type (like `FilesLimitError`) can be used in a `instanceof` expression. I am not sure how to do it.  Checking if the file compiles seems ok, but currently the file doesn't compile because of some other errors.

Reply:
You can simply adding this to the test file.
```ts
expectType<FastifyErrorConstructor>(FilesLimitError);
```